pkg/controller/siddhiprocess/config.go

```diff
@@ -82,7 +82,7 @@ const (
 	MaxSurge                   int32  = 2
 	HealthPort                 int32  = 9090
 	ReadyPrPeriodSeconds       int32  = 20
-	ReadyPrInitialDelaySeconds int32  = 60
+	ReadyPrInitialDelaySeconds int32  = 40
```

Comment:
Can you check whether we can decrease this initial delay more? Does this time includes the pulling time as well?

Reply:
Changed it. These change based on with trial and error method.